docs/training/projects/project-during.md

```diff
@@ -1,42 +1,83 @@
+* All team members are encouraged to contribute here. All ideas and perspectives are needed.
+* Identify any clusters of common tasks or tasks that must be completed sequentially
+* Team members volunteer for tasks that match their personal goals and interests, identify opportunities for working in pairs or small groups
+* Identify if any tasks require skills or knowledge that the current team does not yet have (the project team helper may be able to connect you with someone to provide those skills or knowledge)
+```
+
 ```{image} ../../images/project-taskboard.png
 :alt: example of identifying tasks and organizing them into clusters
 :class: bg-primary mb-1
 :width: 800px
 :align: center
 ```
+### Step 4: Project Management Systems
+
+We're now ready to transfer everything we've done so far into our project management tools!
+
+```{note}
+Set up team organization infrastructure on Slack and GitHub (see [Organizing your project on GitHub](project-github.md)). 
+```
+
+## Project work begins!
+
+Usually teams need at least 1 hour to work through all the steps above. You can think of this as building a strong foundation for the work that happens throughout the rest of the week. 
+
+```{warning}
+It's common for teams to dive into the project work too early. Do you know the names of every person in your group? Have you heard from everyone about their learning goals and ideas for the project tasks? If you answer "no" to any of these, we recommend you take a bit more time working through the activities above. It will pay off for you later in the week!
+```
 
-* Set up team organization infrastructure on Slack and GitHub (see [Organizing your project on GitHub](project-github.md)) 
-* Project work begins!
-    * As progress is made, project outputs (code, notebooks, figures, documentation, etc.) are recorded in the team's GitHub repository (a good place for team members to get started is by making their own folder under the *contributors* directory)
+* As progress is made, project outputs (code, notebooks, figures, documentation, etc.) are recorded in the team's GitHub repository (a good place for team members to get started is by making their own folder under the *contributors* directory).
 * All team members check back in at the end of the day, review progress towards tasks, anything preventing progress, and planned next steps
 * Project leaders may also want to share with the whole hackweek group: How the project is coming along, what are the objectives and tasks identified by the team today, and what other skills might your team need to be successful?
 
-## Days 1 - 5: Project work
+## Daily Team Standups
 
 A "stand up" meeting model may be useful for your team to check in at the beginning and end of each day of the hackweek. During these short meetings, each team member is given a minute or so to briefly share 1) progress they have made on their tasks, 2) anything preventing progress, and 3) what they plan on working on next. 
```

Comment:
```suggestion
A "standup" meeting model may be useful for your team to check in at the beginning and end of each day of the hackweek. During these short meetings, each team member is given a minute or so to briefly share 1) progress they have made on their tasks, 2) anything preventing progress, and 3) what they plan on working on next.

The goal of these "standups" is to make sure everyone has the resources and support they need to achieve their hackweek goals - it can be hard to ask for help when you're working with a new team!
```

@aaarendt I'm sure you can improve the wording on this, but I'm going for reassurance that standups are meant to catch potential pain points early and not leave anyone behind rather than as a tool for measuring if people are accomplishing "enough".